src/games/strategy/triplea/ai/proAI/ProData.java

```diff
@@ -0,0 +1,43 @@
+package games.strategy.triplea.ai.proAI;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.triplea.Properties;
+
+/**
+ * Pro AI data.
+ */
+public class ProData {
+
+  public static double winPercentage = 95;
```

Comment:
There is a big difference between mutable data and immutable data.  Immutable data has properties that make it far less complex and less error prone. For example, immutable data is always thread safe.